ray_julia_jll/gen/make.jl

```diff
@@ -0,0 +1,168 @@
+using Base: BinaryPlatforms
+using CodecZlib: GzipCompressorStream, GzipDecompressorStream
+using LibGit2: LibGit2
+using Pkg
+using Pkg.Types: read_project
+using Tar: Tar
+using URIs: URI
+using ghr_jll: ghr
+
+const ASSETS = Set(["external",
+                    "julia_core_worker_lib.so-2.params",
+                    "_objs",
+                    "julia_core_worker_lib.so.runfiles_manifest",
+                    "julia_core_worker_lib.so",
+                    "julia_core_worker_lib.so.cppmap",
+                    "julia_core_worker_lib.so.runfiles"])
```

Comment:
Do we need anything more than the `.so` file?

Reply:
perhaps not...but we'd have to copy the files out into another tmpdir and then create the tarball from that source.